system/interceptors/SystemAlertsInterceptor.cfc

```diff
@@ -0,0 +1,18 @@
+component extends="coldbox.system.Interceptor" {
+	property name="systemAlertsService" inject="delayedInjector:SystemAlertsService";
+
+	public void function configure() {}
+
+	public void function onApplicationStart() {
+		systemAlertsService.runStartupChecks();
```

Comment:
Could/should this be moved to General.cfc$applicationStart()?

Would be good not to need any interception points here in core given we know that we're going to run this code (so not an extension point really)?